pipeline/console/container/__init__.py

```diff
@@ -310,9 +311,15 @@ def _push_container(namespace: Namespace):
         )
         start_upload_dict = start_upload_response.json()
         upload_token = start_upload_dict.get("bearer", None)
+        true_pipeline_name = start_upload_dict.get("pipeline_name")
 
         if upload_token is None:
             raise ValueError("No upload token found")
+        if true_pipeline_name is None:
```

Comment:
still not quite sure in which instances `true_pipeline_name` will be `None`. Is it just until the latest version of catalyst is deployed?

Reply:
if so, perhaps we don't need this check and can assume it's always present

Reply:
ye its just while we don't have the latest catalyst deployed but people are building and pushing new pipelines. we can remove it if we don't care, but we should just make sure to deploy both changes relatively soon to each other.